service/dap/types.go

```diff
@@ -167,6 +167,10 @@ type LaunchAttachCommonConfig struct {
 	// Default is 50.
 	StackTraceDepth int `json:"stackTraceDepth,omitempty"`
 
+	// Boolean value to indicate whether package paths should be dropped
+	// from function names in the call stack view.
```

Comment:
Replace "function names" with "stack frame names".

I think this is a little less confusing since we already moved the package path away from the stack frame.

Reply:
Sure, see ddbf55d9b3c4.